packages/react-ui/src/widgets/FormulaWidgetUI.js

```diff
@@ -72,26 +67,25 @@ function FormulaWidgetUI(props) {
   const formattedValue = formatter(value);
 
   return (
-    <Box className={classes.root}>
+    <Root>
```

Comment:
I think if you replace this `Root` component with our `Typography`, you don't need this custom styled component:

`<Typography variant='h5' component='div' weight='medium'>`

Reply:
Ok, let me check

Reply:
![image](https://user-images.githubusercontent.com/458196/236898007-e7cf43ac-ba6b-463a-9a9c-822169f8aa23.png)
Change done